src/components/select/Select.tsx

```diff
@@ -1,25 +1,49 @@
+import * as React from 'react';
 import styled from 'styled-components';
 import { colors } from 'common/colors';
+import Icon from 'components/icon/Icon';
 
-const Select = styled.select`
-  transition: all 0.2s;
+interface SelectProps {
+  children: React.ReactNode;
+}
+
+const SelectContainer = styled.div`
+  display: inline;
+  position: relative;
+
+  & > *:last-child {
+    position: absolute;
+    font-size: 1.5rem;
+    top: 0;
+    right: 0;
+    pointer-events: none;
+    user-select: none;
+    color: ${colors.grayLighten60};
+  }
+`;
 
-  padding: 8px 28px 8px 8px;
+const StyledSelect = styled.select`
+  transition: all 0.2s;
+  padding: 8px 20px 8px 8px;
   background: #fff;
   border-radius: 3px;
-  border: 1px solid ${colors.grayLighten90};
-  box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.2);
+  border: 2px solid ${colors.grayLighten60};
```

Comment:
![](https://folk.ntnu.no/johannkv/img/wn5rth1IXY.png)
For some reason, 1px of some longer letters disappear into the border here.
This could also be a problem with the text styling in general tho.

Reply:
This is super weird, never seen this before. It's just a normal border, so it shouldn'ẗ misbehave. Won't fix until it becomes a problem though - you should never want a select right next to something without margin anyways